arviz/stats/density_utils.py

```diff
@@ -566,7 +566,7 @@ def _kde_linear(
     x_len = len(x)
```

Comment:
```suggestion

```

Reply:
Merged this in and squashed the commits into one.